packages/models-library/src/models_library/services_metadata_editable.py

```diff
@@ -21,6 +21,7 @@ class ServiceMetaDataEditable(ServiceBaseDisplay):
     name: str | None  # type: ignore[assignment]
     thumbnail: HttpUrl | None
     description: str | None  # type: ignore[assignment]
+    description_ui: bool = False
```

Comment:
@sanderegg I understand your point, specially regarding the redundancy. I also asked myself how is it possible adding a single field would require so many changes. Nonetheless, a careful looks reveals that there is a purpose behind (more details later)

> so many defaults in services.py, services_base.py and here. will this not bite us?
No, i do not think so.  Here the default is added to make it optional

The same model structure (i.e. collection of fields) need different setups (e.g. optional/required/nullable) depending on whether they are used as input or output models in the same API. In addition, the very same fields are moved from one api to another. That is what produces the redundancy. It is indeed very redundant but given the tech stack we have I believe is the most reliable way to debug and even understand in case there are issues. Nonetheless, I am happy to review these in case you/someone has some ideas.